static/src/js/util/isLoggedIn.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns true if the authToken is not empty
+ * @return {boolean}
```

Comment:
This jsdoc should list the parameter `@param {Type} paramName Param description`